proto/frequenz/api/common/v1/metrics/metrics.proto

```diff
@@ -172,6 +172,49 @@ enum Metric {
   METRIC_SENSOR_IRRADIANCE = 166;
 }
 
+// Enumerated categories of connections from which metrics can be obtained for a
+// given electrical component or sensor.
+enum MetricConnectionCategory {
+  // Default value.
+  METRIC_CONNECTION_CATEGORY_UNSPECIFIED = 0;
+
+  // Represents a connection to a battery.
+  METRIC_CONNECTION_CATEGORY_BATTERY = 1;
+
+  // Represents a connection to a PV array.
+  METRIC_CONNECTION_CATEGORY_PV_ARRAY = 2;
+
+  // Represents a connection to a temperature sensor measuring the ambient
+  // temperature.
```

Comment:
Do we want to be so specific? Can this be for something else than temperature? Like humidity, dew point, etc.? Same for CABINET, HEATSINK, etc.